dotcom-rendering/src/types/trails.ts

```diff
@@ -37,12 +37,16 @@ export interface TrailType extends BaseTrailType {
 
 export interface CAPITrailType extends BaseTrailType {
 	format: CAPIFormat;
-	// Include pillar and designType until we remove them upstream
-	// We type designType as `string` for now so that the field is present,
-	// but we don't care what's in it. Pillar we have a type for so we use it
-	// but it shouldn't be important.
+	/**
+	 * @deprecated This type must exist as it's passed by frontend, but we shouldn't use it.
+	 * We should remove this properties upstream in the future
```

Comment:
I wouldn't normally be such a pedant, but the pluralisation makes it confusing in the sense of I was looking for multiple fields

```suggestion
	 * We should remove this property upstream in the future
```